__tests__/app.test.js

```diff
@@ -1,69 +1,221 @@
+                expect(articles.length).toBe(13);
+                articles.forEach((article) => {
+                    expect(article).toMatchObject({
+                        title: expect.any(String),
+                        topic: expect.any(String),
+                        author: expect.any(String),
+                        created_at: expect.any(String),
+                        votes: expect.any(Number),
+                        article_img_url: expect.any(String),
+                        comments_count: expect.any(String),
+                    });
+                    const articleBody = { body: expect.any(String) };
+                    expect(articles).toEqual(
+                        expect.not.objectContaining(articleBody)
+                    );
+                    expect(articles).toBeSorted({ descending: true });
+                });
+            });
     });
 });
 
-describe('/api/articles/:article_id', () => {
-    test('GET: 200 an article object with all associated properties', () => {
+describe("GET: /api/articles/:article_id/comments", () => {
+    test("GET: 200 returns all comments associated with a specific article_id in ascending order based on creation date", () => {
         return request(app)
-        .get('/api/articles/1')
+            .get("/api/articles/1/comments")
+            .expect(200)
+            .then((response) => {
+                const { comments } = response.body;
+                expect(comments.length).toBe(11);
+                expect(comments).toBeSorted({ descending: false });
+                expect(comments[0]).toMatchObject({
+                    comment_id: 9,
+                    body: "Superficially charming",
+                    article_id: 1,
+                    author: "icellusedkars",
+                    votes: 0,
+                    created_at: "2020-01-01T03:08:00.000Z",
+                });
+            });
+    });
+});
+test("GET: 200 Article does not have any comments", () => {
+    return request(app)
+        .get("/api/articles/13/comments")
         .expect(200)
         .then((response) => {
-            const {rows} = response.body
-            rows.forEach(row => {
-                expect(row).toMatchObject({
-                    title: expect.any(String),
-                    topic: expect.any(String),
-                    author: expect.any(String),
-                    body: expect.any(String),
-                    created_at: expect.any(String),
-                    votes: expect.any(Number),
-                    article_img_url:
-                    expect.any(String),
-                })
-            })
-        })
-    })
-});
\ No newline at end of file
+            expect(response.body.msg).toBe("No comments yet");
+        });
+});
+test("POST:201 posting a new comment", () => {
+    const newComment = {
+        author: "butter_bridge",
+        comment: "did  you know that peppa pig is actually vegetarian?",
+        article_id: 1,
```

Comment:
Ideally we don't want the article_id in the body, this should be coming from the end point itself ( /api/articles/1/comments)